src/main/java/teammates/common/datatransfer/FeedbackConstantSumQuestionDetails.java

```diff
@@ -471,6 +473,44 @@ public String getQuestionResultStatisticsCsv(
         }
         return optionPoints;
     }
+    
+    /**
+     * Used to update the option points mapping with a default value when distributing to recipients
+     * @param optionPoints
+     * @param question question for the option points mapping
+     * @param responses responses for the option points mapping
+     * @param bundle
+     */
+    private void updateRecipientOptionPointsMappingWithDefaultValue(
```

Comment:
Maybe add a comment, or an additional parameter, for the value of the default value. It will make thinking about this question type and debugging easier in future.